types/slate-html-serializer/index.d.ts

```diff
@@ -1,20 +1,21 @@
 import * as React from "react";
 import { BlockProperties, Leaf, Mark, Node as SlateNode, Value, ValueJSON } from "slate";
 
-export interface Rule {
+interface Rule {
     deserialize?:
         | ((el: Element, next: (elements: Element[] | NodeList | Array<Node & ChildNode>) => any) => any)
         | undefined;
     serialize?: ((obj: any, children: string) => React.ReactNode) | undefined;
 }
 
-export interface HtmlOptions {
+interface HtmlOptions {
     rules?: Rule[] | undefined;
     defaultBlock?: BlockProperties | string | undefined;
     parseHtml?: ((html: string) => HTMLElement) | undefined;
 }
 
-export default class Html {
+declare class Html {
```

Comment:
This package exports the class using default. So this is incorrect change.

Reply:
Ah good call! Will fix and add test

Reply:
Hm, does it? The bottom of https://unpkg.com/browse/slate-html-serializer@0.6.32/lib/slate-html-serializer.js has `module.exports = Html`. Of course, this is also funky because they also have a `"module"` in `package.json` but that's not reachable via ESM imports, only bundlers.

(Also, upstream is at 0.8, and the types are at 0.6, though they seem to be the same.)

Reply:
I'm dumb; it very clearly says `exports.default`. Nevermind.

Reply:
Oh wait I misread this and added a test for the wrong thing

Reply:
I think this is still correct as is. See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68135#discussion_r1445243547

Reply:
Also the default import is covered by existing tests: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/slate-html-serializer/slate-html-serializer-tests.ts#L12